### PR TITLE
feat(details): introduce accordion

### PR DESCRIPTION
**Pull request by kishore03109**
Accordion styles (currently live on staging for the `kishore-test` repo)
Screenshot 2023-12-06 at 4 46 48 PM

### CODE REVIEW

File: _sass/components/_details.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  }
+}
+
+details.isomer-details summary::marker {
+  display: none;
+}
+
+details.isomer-details summary::after {
+  // TODO: Think of a better way to do this
+  content: url("https://raw.githubusercontent.com/atisawd/boxicons/master/svg/solid/bxs-chevron-down.svg");
```
Comment (seaerchin) [inline _sass/components/_details.scss]: if we are set on using boxicons (think we are), we can just link the script and use the web component `box-icon`

File: _sass/components/_details.scss [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  border-bottom: 1px solid #d4d4d4;
+}
+
+.isomer-accordion-white > details.isomer-details > summary {
+  background-color: #fff;
+  color: #4372d6;
+}
+
+.isomer-accordion-gray > details.isomer-details > summary {
+  background-color: #f4f4f4;
```
Comment (seaerchin) [inline _sass/components/_details.scss]: nit: can put these in colours

File: _sass/components/_details.scss (hunk 1 above)
Comment (seaerchin) [reply _sass/components/_details.scss]: cc @sehyunidaaa

Comment (kishore03109) [reply _sass/components/_details.scss]: could you explain further ah @seaerchin? exactly what source link are you referring to ah

@sehyunidaaa can confirm using [bi-chevron-down](https://react-icons.github.io/react-icons/search/#q=bichevrondown)

File: _sass/components/_details.scss (hunk 2 above)
Comment (kishore03109) [reply _sass/components/_details.scss]: hey @seaerchin ok to put in a separate pr? some tokens not defined in yet

File: _sass/components/_details.scss (hunk 1 above)
Comment (sehyunidaaa) [reply _sass/components/_details.scss]: ooh my this is a long time ago. that's fine, but why not use the `sgds-icon sgds-icon-chevron-down` that we already have in the template? so we can keep it to one icon style

**Comment by seaerchin** [comment]
please ping me or re-request when ready. idt there's much work left irght?

File: _sass/components/_details.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    justify-content: flex-end;
+  }
+}
+
+details.isomer-details summary::marker {
+  display: none;
+}
+
+details.isomer-details summary::after {
+  content: "\E93C";
```
Comment (seaerchin) [inline _sass/components/_details.scss:18]: could we add a comment here

File: _sass/components/_details.scss (hunk 2 above)
Comment (seaerchin) [reply _sass/components/_details.scss]: @sehyunidaaa - do we have the tokens?

**Review by seaerchin — APPROVED** [review]
skimmed; no real blockers apart from comments + colours. please do prior to merge as these are unlikely to change but will be a source of confusion later on/add extra work

**Pull request merged**